examples/version-migration/schema-upgrade/src/modelVersion1/appModel.ts

```diff
@@ -61,7 +60,7 @@ export class InventoryListAppModel implements IInventoryListAppModel, IMigratabl
 		return `version:one\n${inventoryItemStrings.join("\n")}`;
 	};
 
-	public close() {
-		this.container.close();
+	public dispose() {
```

Comment:
This seems like a breaking change of sorts

Reply:
I suppose folks could implement IMigratableModel.dispose() differently here if they want (i.e. have it call container.close() instead if they're supporting old loader).

Should just follow the exact same pattern as whatever they do today for deciding when to call container.close() vs. container.dispose() in their non-migration scenarios.